tests/runtest.cmd

```diff
@@ -432,29 +432,36 @@ exit /b 0
 :Usage
 echo.
 echo Usage:
-echo   %0 BuildArch BuildType [TestEnv TEST_ENV_SCRIPT] [VSVersion] CORE_ROOT
+echo   %0 [options] [CORE_ROOT]
```

Comment:
My understanding of the typical cmd line convention is that `CORE_ROOT` should be in `< >` to indicate it is the name of an argument, not the value, corresponding to below.